src/legacy/ui/public/vislib/visualizations/_labels.scss

```diff
@@ -0,0 +1,22 @@
+$visBarLabelDarkColor: $euiColorDarkShade;
+$visBarLabelLightColor: $euiColorLightShade;
+
+.bar-label--normal {
+  font-size: 8pt;
```

Comment:
Don't use `pt` measures here. You can instead use our EUI variable of `$euiSizeS` for `8px`

Reply:
`$euiSizeS` = `8px` is too small.
![image](https://user-images.githubusercontent.com/8614638/59739074-1e100380-926c-11e9-8256-42c8176d3b66.png)

I set `8pt`, because i try to align size with axis label which hardcoded with `8pt` in `axis_config.js (57)`:
```
    color: '#ddd',
    font: '"Open Sans", "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif',
    fontSize: '8pt',
    truncate: 100
  },
```
Maybe i should get the size from this config like in axis_label.js by:
```
config.get('labels.fontSize')
```
But then, it will excluded from css classes.

Reply:
Does it pull the other style info from there like the font family?

Reply:
I'm not sure what do you mean by 'it'. By now the i only set the fontSize to 8px as like in axis label, not the font family. Maybe axis label should also factor out (at least defaults) to css classes, if so, we can share the same classes both to axis and bar labels.

Reply:
Ok, just leave it in the CSS as `8pt` for now. We'll figure out the best way to consolidate those styles later. Thanks!